presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -133,7 +133,7 @@ export class PresentationRpcInterface extends RpcInterface {
   public static readonly interfaceName = "PresentationRpcInterface"; // eslint-disable-line @typescript-eslint/naming-convention
 
   /** The semantic version of the interface. */
-  public static interfaceVersion = "2.8.0";
+  public static interfaceVersion = "2.9.0";
 
```

Comment:
Why did the interface version change?

Reply:
I was thinking about this... Bumping the version prevents us from the situation where newer frontends use older backends. The new rules that I added are sent to the backend for handling, so without bumping the version we'd be in a situation where they are sent and not handled. That wouldn't break anything, since the new attributes are optional, but the result wouldn't be exactly what you'd expect with the given ruleset. So I wanted to avoid that.